packages/common/pipes/file/parse-file.pipe.ts

```diff
@@ -25,13 +27,20 @@ export class ParseFilePipe implements PipeTransform<any> {
       exceptionFactory,
       errorHttpStatusCode = HttpStatus.BAD_REQUEST,
       validators = [],
+      fileIsRequired = true,
     } = options;
 
     this.exceptionFactory =
       exceptionFactory ||
       (error => new HttpErrorByCode[errorHttpStatusCode](error));
 
     this.validators = validators;
+
+    this.validators.unshift(
+      new RequiredFileValidator({
+        required: fileIsRequired,
+      }),
+    );
```

Comment:
can't we have just

```suggestion
    if (fileIsRequired)
      this.validators.unshift( new RequiredFileValidator() );
```

as the name of the validator is 'required'

Reply:
In the end, I had to completely remove the validator. The problem here was that this validator could validate an incoming optional file, but if there were any other validators expecting a file to exist, the error would be thrown again. In conclusion, I decided to go back to the previous implementation, only changing the interface to use the `isFileRequired` option.